### PR TITLE
sharing code changed to be in sharing.mjs. Reformatted topbar to better allow for dual-row topbar if we decide to do that.

### PR DESCRIPTION
**Pull request by MichaelEbert**

### CODE REVIEW

File: template/navbar.html [hunk 1]
```diff
@@ -1,45 +1,48 @@
...
-	<div class="topbarSection icon">
-		<a href="https://prclive.run/"><img src="images/prcbanner.png" style="width: 90px;"></a>
-		<div class="topbarSublist">
-			<a href="https://www.twitch.tv/prclive" class="topbarSection">Twitch</a>
-			<a href="https://www.youtube.com/c/prclive" class="topbarSection">YouTube</a>
-			<a href="https://discord.gg/eudsps7" class="topbarSection">Discord</a>
-			<a href="https://github.com/MichaelEbert/OblivionProgressTracker" class="topbarSection">GitHub</a>
-			<a href="https://twitter.com/_prclive" class="topbarSection">Twitter</a>
+<div id="topbar" class="topbar">
+	<div id="sharingBanner"></div>
```
Comment (MeemawHustlin) [inline template/navbar.html:3]: this is on all pages but is never used

File: css/main.css [hunk 2]
```diff
@@ -84,22 +84,26 @@ kbd {
...
 	position: fixed;
 	background-color: var(--base-color);
-	padding: 0.3em;
-	box-sizing: border-box;
 	top: 0; /* hug the top of the window instead of having a buffer */
 	left: 0;
 	z-index: 1;/*Lets topbar render ontop of iframe*/
 }
 
+.topbarNav{
```
Comment (MeemawHustlin) [inline css/main.css:97]: something in here has broken the navbar's ability to stack its options when the browser width gets smaller so now you can't actually get to the guide and PRC tabs after a certain point.

**Review by MeemawHustlin — CHANGES_REQUESTED** [review]
not entirely sure what the new topbarNav benefits are since the last commit removed all the functionality tied to it.

**Comment by MichaelEbert** [comment]
Purpose of this now is mainly code refactor and structural changes for future commits. Make future diffs less painful.

File: css/main.css (hunk 2 above)
Comment (MichaelEbert) [reply css/main.css:97]: Ty for catching this. Will take a look.

Comment (MichaelEbert) [reply css/main.css:97]: Fixed. Was because we have some narrow-screen-only formatting that I didn't change before.

**Review by MeemawHustlin — APPROVED** [review]

**Pull request merged**